src/init/common.h

```diff
@@ -8,11 +8,11 @@
 #ifndef BITCOIN_INIT_COMMON_H
 #define BITCOIN_INIT_COMMON_H
 
+#include <kernel/init/common.h> // IWYU pragma: export
+
 class ArgsManager;
 
 namespace init {
-void SetGlobals();
-void UnsetGlobals();
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/25065#discussion_r882023913

> Doing that does mean we can't remove `compat/glibcxx_sanity.cpp` from libbitcoinkernel

Looking at current `SanityChecks()` function, maybe a bigger problem moving it into the kernel is that it is using
`InitError` from  `ui_interface.cpp`. So might be worth postponing this or doing it separately if it would complicate this PR.

Reply:
Fixed as of 9ac87854af62d271df2f383d8f2f798c70453448

Reply:
Nice, thanks.

Now that we have the notion of a context, it would be possible to either:
- Make it impossible to create an invalid context (sanity checks called at construction time)
- Allow for "invalid" contexts, which api calls check for.

But those can be done much later when we start actually defining an api.